### ISSUE TITLE
Schema validation errors not caught by error handler

### ISSUE THREAD

**Issue by NMFR** [zalando/connexion#1016]
### Description

Schema validation errors are not caught by error handlers.
I want my application to have a "standard" JSON error format.
For this i register an error handler to format the error response.
Schema validation errors bypass the error handler and do not allow to format the response.

### Expected behaviour

```
HTTP 500
{ "error": 'My custom error format" }
```

### Actual behaviour

```
HTTP 400
{
  "detail": "1000 is greater than the maximum of 90",
  "status": 400,
  "title": "Validation error",
  "type": "about:blank"
}
```

### Steps to reproduce

application.py:
``` python
connexion_app = connexion.App(root_path, specification_dir='src/api/rest')

def handle_connexion_exception(exception):
        return '{ "error": 'My custom error format" }', 500

connexion_app.add_error_handler(ProblemException, handle_connexion_exception)
```

Cause a openAPI schema error (call an endpoint with invalid JSON).

### Additional info:

The culprit seems to be `validate_schema` https://github.com/zalando/connexion/blob/master/connexion/decorators/validation.py#L188 that does not throw an error and just returns the endpoint response.

**Comment by dtkav** [comment]
I think this is fixed by #955 
Can you try the latest release?

**Comment by NMFR** [comment]
Fixed
